models-info/src/main/java/com/flipkart/fdp/ml/transformer/CommonAddressFeaturesTransformer.java

```diff
@@ -0,0 +1,61 @@
+package com.flipkart.fdp.ml.transformer;
+
+import com.flipkart.fdp.ml.modelinfo.CommonAddressFeaturesModelInfo;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CommonAddressFeaturesTransformer implements Transformer {
+	private CommonAddressFeaturesModelInfo modelInfo;
+	private final Pattern pattern = Pattern.compile("[0-9]");
```

Comment:
we can rename pattern to something like is_digit_pattern

Reply:
renamed